src/assets/scss/app.scss

```diff
@@ -899,3 +899,11 @@ becomes more complicated.
   transition: opacity 0.6s 0.15s;
 }
 [data-mark-rows-deleted="false"] { display: none; }
+
+
+////////////////////////////////////////////////////////////////////////////////
+// Don't let styles defined in Central bleed into new Web Forms Component
+
+.odk-form :not(svg|*) {
```

Comment:
I feel like it'd be better to move this to `FormPreview`, since that's where all the other Web Form setup lives. `FormPreview` already houses some other Web Form-related CSS.

Reply:
I tried that but doesn't work because styles defined in the SFC are loaded after style coming from `web-form` component, in that case this rule reverts the `web-form` styles as well.

Reply:
Ah very interesting, that makes sense. :+1: